tools/unity.BUILD

```diff
@@ -0,0 +1,9 @@
+cc_library(
```

Comment:
See
https://docs.bazel.build/versions/main/be/c-cpp.html#cc_library.includes
https://docs.bazel.build/versions/main/be/c-cpp.html#cc_library.include_prefix
https://docs.bazel.build/versions/main/be/c-cpp.html#cc_library.strip_include_prefix
and mess around so that you can include unity with <>. This is important because bazel will treat it as a system dependency and not subject it to the many compiler warnings that you will want to add.

I would recommend `#include <third_party/unity/unity.h>`

Reply:
Also consider moving this from `tools` to a new top-level `third_party` directory.